common/app/model/LiveBlogCurrentPage.scala

```diff
@@ -70,6 +74,18 @@ object LiveBlogCurrentPage {
         BlockPage(blocks = Nil, blockId = blockId, pageNumber = numPages, filterKeyEvents)
       }
 
+      // returns latest pinned post and blocks minus the pinned post
```

Comment:
Consider putting this in a method called something like removeFirstBlockIfPinned? instead of adding a comment.
If you do, identifying the pinned post should happen outside of this method (lines 79&80)